functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configuration/SpecificTestClassExecutionFunctionalTest.java

```diff
@@ -0,0 +1,140 @@
+package org.arquillian.smart.testing.ftest.configuration;
+
+import java.util.Collection;
+import org.arquillian.smart.testing.ftest.testbed.project.Project;
+import org.arquillian.smart.testing.ftest.testbed.rules.GitClone;
+import org.arquillian.smart.testing.ftest.testbed.rules.TestBed;
+import org.arquillian.smart.testing.ftest.testbed.testresults.TestResult;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.arquillian.smart.testing.ftest.testbed.configuration.Mode.ORDERING;
+import static org.arquillian.smart.testing.ftest.testbed.configuration.Mode.SELECTING;
+import static org.arquillian.smart.testing.ftest.testbed.configuration.Strategy.AFFECTED;
+import static org.arquillian.smart.testing.ftest.testbed.configuration.Strategy.NEW;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SpecificTestClassExecutionFunctionalTest {
+
+    @ClassRule
+    public static final GitClone GIT_CLONE = new GitClone();
+
+    @Rule
+    public TestBed testBed = new TestBed(GIT_CLONE);
+
+    private static final String module = "config/impl-base";
+
+    private static final String EXPECTED_LOG_PART = "Enabling Smart Testing";
+
+    @Test
+    public void should_disable_extension_and_only_execute_specified_test_when_single_test_set() throws Exception {
+        // given
+        final Project project = testBed.getProject();
+
+        project.configureSmartTesting()
+                .executionOrder(NEW)
+                .inMode(ORDERING)
+            .enable();
+
+        final Collection<TestResult> expectedTestResults = project
+            .applyAsCommits("Adds new unit test");
+
+        // when
+        final Collection<TestResult> actualTestResults = project
+            .build(module)
+                .options()
+                    .withSystemProperties("scm.range.head", "HEAD", "scm.range.tail", "HEAD~", "test", "PropertiesParserTestCase", "failIfNoTests", "false")
+                .configure()
+            .run("clean", "test");
+
+        // then
+        String capturedMavenLog = project.getMavenLog();
+        assertThat(capturedMavenLog).doesNotContain(EXPECTED_LOG_PART);
```

Comment:
The tool isn't feasible for us because we do not directly get the instance of logger. In our test-bed, we get execute the main classes from the embedded maven and thus instance of the logger somewhere gets lost and we get null values.
So, we'll have to use maven logs only.